rosdep/base.yaml

```diff
@@ -1061,6 +1061,9 @@ geos:
   fedora: [geos-devel]
   gentoo: [sci-libs/geos]
   ubuntu: [libgeos-dev]
+gettext-base:
+  debian: [gettext-base]
+  ubuntu: [gettext-base]
```

Comment:
Can you please also add a key for Fedora?  I believe it will be: `fedora: [gettext]`.